content/en/synthetics/private_locations/_index.md

```diff
@@ -211,6 +211,24 @@ This command starts a Docker container and makes your private location ready to
 
 [1]: https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities
 
+{{% /tab %}}
+
+{{% tab "Podman" %}}
+
+The Podman configuration is very similar to Docker, however, you must set `NET_RAW` as an additional capability to support ICMP tests. 
+
+1. Run `sysctl -w net.ipv4.ping_group_range = 0 2147483647` from the host where the container runs.
+2. Run this command to boot your private location worker by mounting your configuration file to the container. Ensure that your `<MY_WORKER_CONFIG_FILE_NAME>.json` file is accessible to mount to the container:
+   
+   ```go
+   podman run –cap-add=NET_RAW --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest
```

Comment:
```suggestion
   ```shell
   podman run --cap-add=NET_RAW --rm -it -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json gcr.io/datadoghq/synthetics-private-location-worker:latest
```